src/MainPage.jsx

```diff
@@ -242,10 +255,15 @@ class App extends Component {
     // store the pending PaymentIntent's secret until the payment is complete.
     if (!this.pendingPaymentIntentSecret) {
       try {
+        let paymentMethodTypes = ["card_present"];
+        if (this.state.currency === "cad") {
+          paymentMethodTypes.push("interac_present");
+        }
```

Comment:
There was implementation review feedback that these three lines were not working properly -- could you confirm that they are working given the new backend change?

Reply:
yea, I'm going to merge the example backend and test this with the new backend.